src/m1/team14/view/AbstractViewPanel.java

```diff
@@ -0,0 +1,8 @@
+package m1.team14.view;
```

Comment:
commonly,
the name of Interface is like IViewPanel
the name of Abstract class is like AbstractViewPanel